dashboard/top-sql.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Top SQL
+summary: This document describes how to use Top SQL to locate SQL queries that contribute to a high load.
+---
+
+# Top SQL
+
+> **Warning:**
+>
+> Currently, Top SQL is an experimental feature. It is not recommended that you use it for production environments.
+
+This document describes how to use Top SQL to locate SQL queries that contribute to a high load of a TiDB or TiKV node in a specified time range. For example, you can use Top SQL to locate an analytic query that contributes to 99% of the load for a low-load database.
+
+For a specified TiDB or TiKV node, Top SQL provides the following features:
+
+* Show the top 5 types of SQL queries that contribute the most to the load in a specified time range.
+* Show information such as CPU usage, requests per second, average latency, and query plan of a particular query, which can be used for potential performance optimization to improve your business.
+
+## Enable Top SQL
+
+The Top SQL feature is disabled by default. You can enable the feature for the entire cluster using either of the following methods:
+
+- Method 1: Log in to TiDB Dashboard, click **Top SQL** in the left pane, click the gear button in the upper-right corner of the page, and then enable the Top SQL feature.
+- Method 2: Set the value of the TiDB system variable [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-introduced-from-v540-version) to `ON`.
+
+> **Note:**
+>
+> Enabling Top SQL has a slight impact on cluster performance.
+
+## Use Top SQL
+
+Once Top SQL is enabled, you can log into TiDB Dashboard, and then click **Top SQL** in the left pane to use it.
+
+![Top SQL](/media/dashboard/top-sql-overview.png)
+
+Usage tips：
+
+* You can select the target node and time range in the drop-down list at the top of the page, or you can select the time range in the chart.
+* If the data in the chart is out of date, you can click **Refresh**, or select auto-refresh and specify the auto-refresh interval in the **Refresh** drop-down list.
```

Comment:
```suggestion
* If the data in the chart is out of date, you can click **Refresh** button, or select auto-refresh and specify the auto-refresh interval in the **Refresh** drop-down menu.
```

Reply:
@SunRunAway 
**Refresh** does not look like a button because it has a drop-down arrow on the right, how about I change this paragraph as follows?

```suggestion
* If the data in the chart is out of date, you can click **Refresh** at the top of the page, or click the drop-down arrow beside **Refresh** to specify an auto-refresh interval.
```

Reply:
The **Refresh**  is conbined with a button and a drop-down menu
So when the users want to refresh, they can click the **Refresh** button, or they can chooss the auto-refresh interval in the **Refresh** drop-down menu.

Reply:
If we mention both **Refresh** button and **Refresh** drop-down menu in documents, it might give users an impression that there are two different components in different locations, so I think it is better to keep the original content.